dspace-spring-rest/src/test/java/org/dspace/app/rest/GroupRestRepositoryIT.java

```diff
@@ -29,6 +32,40 @@
 
 public class GroupRestRepositoryIT extends AbstractControllerIntegrationTest {
 
+    @Test
+    public void createTest()
```

Comment:
This test looks good, but it looks to only provide a test with a valid authentication/authorization.  We should also test that you *cannot* post a new group if you are not an Administrator and/or are not logged in.

For reference, this PR #2228 is implementing similar tests (except for Community creation).  But, it might be used as an example.

Reply:
The test are already implement. You will be find it from line 59 - 66.

Reply:
@antzsch : I would recommend splitting these two tests out, as I completely overlooked that secondary test.  For example, have a `createTest()` which tests a valid creation, and a `createUnauthorizedTest()` which tests the scenarios of being logged in as a non-admin, and also being anonymous.